colour/models/rgb/transfer_functions/exponent.py

```diff
@@ -0,0 +1,290 @@
+        return np.where(x >= 0, exponent_forward(x), -exponent_forward(-x))
+    elif style == 'basicmirrorrev':
+        return np.where(x >= 0, exponent_reverse(x), -exponent_reverse(-x))
+    elif style == 'basicpassthrufwd':
+        return np.where(x > 0, exponent_forward(x), x)
+    elif style == 'basicpassthrurev':
+        return np.where(x > 0, exponent_reverse(x), x)
+    else:
+        raise ValueError(
+            'Undefined style used: "{0}", must be one of the following: '
+            '"{1}".'.format(
+                style, ', '.join([
+                    'basicFwd', 'basicRev', 'basicMirrorFwd', 'basicMirrorRev',
+                    'basicPassThruFwd', 'basicPassThruRev'
+                ])))
+
+
+def exponent_function_monitor_curve(x, exponent, offset, style='monCurveFwd'):
+    """
+    Defines the *Monitor Curve* exponent transfer function.
+
+    Parameters
+    ----------
+    x : numeric or array_like
+        The data to undergo monitor curve exponential transfer.
+    exponent : numeric or array_like
+        The exponent value used for the transfer.
+    offset: numeric or array_like
+        The offset value used for the transfer.
+    style : unicode, optional
+        **{'monCurveFwd', 'monCurveRev', 'monCurveMirrorFwd',
+        'monCurveMirrorRev'}**,
+        Defines the behaviour for the transfer function to operate:
+
+        -   *monCurveFwd*: *Monitor Curve Forward* exponential behaviour
+            where the function definition applies a power law function
+            with a linear segment near the origin.
+
+        -   *monCurveRev*: *Monitor Curve Reverse* exponential behaviour
+            where the function definition applies a power law function
+            with a linear segment near the origin.
+
+        -   *monCurveMirrorFwd*: *Monitor Curve Mirror Forward*
+            exponential behaviour where the function definition applies
+            a power law function with a linear segment near the origin
+            and mirrors the function for values less than zero
+            (i.e. rotationally symmetric around the origin).
+
+        -   *monCurveMirrorRev*: *Monitor Curve Mirror Reverse*
+            exponential behaviour where the function definition applies
+            a power law function with a linear segment near the origin
+            and mirrors the function for values less than zero
+            (i.e. rotationally symmetric around the origin).
+
+    Returns
+    -------
+    numeric or ndarray
+        *Monitor curve* exponentially transferred data.
+
+    Raises
+    ------
+    ValueError
+        If the *style* of function operation is not defined.
+
+    Examples
+    --------
+    The monitor curve exponent function *styles* operate as follows:
+
+    >>> exponent_function_monitor_curve(2, 2, 2)
+    array(1.7777777777777777)
+    >>> exponent_function_monitor_curve(-2, 2, 2)
+    array(-1.7777777777777777)
+    >>> exponent_function_monitor_curve(1.7777777777777777, 2, 2,
```

Comment:
I haven't looked at the full PR yet but builds are breaking because you need an ellipsis here, direct line return break the parsing, so you need to do something like that:

```python
    >>> exponent_function_monitor_curve(-1.7777777777777777, 2, 2,
    ...                                 'monCurveRev')
```
or preferred:

```python
    >>> exponent_function_monitor_curve(
    ...    -1.7777777777777777, 2, 2 'monCurveRev')
```

Reply:
Noted. Have made the changes, and will take care of this in the future! :)